service/build.gradle

```diff
@@ -132,6 +133,7 @@ tasks.withType(Test) {
 }
 
 check.dependsOn integrationTest
+jacocoTestReport.dependsOn integrationTest
```

Comment:
Is this not what's causing the integration tests to run as part of the unit tests?